submissions/TatyanaMolchanova/js-dom/style.css

```diff
@@ -0,0 +1,66 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+    font-family: Arial;
+    font-size: 10px;
+}
+
+.content {
+    display: flex;
+}
+
+.aside {
+    min-width: 30%;
+}
+
+.menu {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+}
+
+.main {
+    padding: 15px;
+    max-width: 70%;
+    overflow: hidden; 
+}
+
+.menu__item {
```

Comment:
Please, add `cursor: pointer;` for this element or just use `a` tag inside `li` tag

Reply:
I added cursor: pointer; to menu item.